tests/e2e/test_artifacts/generated_e2e_test_fetch_models.cmake

```diff
@@ -0,0 +1,131 @@
+# Fetch the model from "https://storage.googleapis.com/iree-model-artifacts/deeplabv3.tflite"
```

Comment:
I think these comments are pretty redundant

Reply:
Let's not touch unrelated files in this change : ) I'll do it in another PR.